### PR TITLE
fix(prowjob): fix command link to caicloud

### PR DESCRIPTION
**Pull request by bbbmj**

### CODE REVIEW

**Comment by bbbmj** [comment]
@ddysher @zjj2wry @supereagle PTAL

**Review by supereagle — APPROVED** [review]

**Comment by deyuanix** [comment]
let's keep it open before we figure out our doc location

File: prow/plugins/approve/approvers/owners.go [hunk 1]
```diff
@@ -585,7 +585,7 @@ Associated issue requirement bypassed by: {{range $index, $approval := .ap.ListN
 
 {{ end -}}
 
-The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands).
+The full list of commands accepted by this bot can be found [here](https://github.com/caicloud/engineering/getting-started/bot-commands.md).
```
Comment (deyuanix) [inline prow/plugins/approve/approvers/owners.go]: can you update this? i think we've settled down with our doc location.

Comment (deyuanix) [reply prow/plugins/approve/approvers/owners.go]: @bbbmj

**Comment by deyuanix** [comment]
@bbbmj ping?

File: prow/plugins/approve/approvers/owners.go (hunk 1 above)
Comment (bbbmj) [reply prow/plugins/approve/approvers/owners.go]: ok, I will make it later

**Comment by bbbmj** [comment]
updated! @ddysher

**Pull request merged**